brave/browser-laptop
# only update the menu when bookmarks or closed tabs change

Test Plan:
Add bookmark
1. Go to brave.com
2. Add bookmark
3. Bookmark should appear in menu

Bookmark indicator
1. Open a new tab
2. Bookmark indicator should not be activated
3. Bookmarks menu "Bookmark this page" should not be checked
4. Go back to brave.com tab
5. Bookmark indicator should be active
6. Bookmarks menu "Bookmark this page" should be checked

Last closed Frame
1. Close the brave.com tab
2. History menu should show brave.com under recently closed

Language
1. Change language
2. Restart Brave
3. Text should be translated by language setting
4. Change back and restart Brave
5. Text should be translated by language setting

State
1. Open brave.com
2. Close Brave
3. Reopen Brave and cnn.com tab should be loaded

Fix #3782
